homeassistant/components/knx/schema.py

```diff
@@ -84,9 +86,10 @@ class BinarySensorSchema:
 
     CONF_STATE_ADDRESS = CONF_STATE_ADDRESS
     CONF_SYNC_STATE = CONF_SYNC_STATE
+    CONF_INVERT = CONF_INVERT
     CONF_IGNORE_INTERNAL_STATE = "ignore_internal_state"
     CONF_CONTEXT_TIMEOUT = "context_timeout"
-    CONF_RESET_AFTER = "reset_after"
+    CONF_RESET_AFTER = CONF_RESET_AFTER
```

Comment:
```suggestion
```

Reply:
This has been done for consistency reasons and would break https://github.com/home-assistant/core/blob/dev/homeassistant/components/knx/factory.py#L250